storage/redistore/peer_store.go

```diff
@@ -0,0 +1,169 @@
+//Note: ip6 seperation into shards is unnecessary when using Redis(?)
+package redistore
+
+import (
+	"encoding/binary"
+	"log"
+	"time"
+
+	"github.com/chihaya/chihaya/bittorrent"
+	"github.com/chihaya/chihaya/storage"
+	"github.com/garyburd/redigo/redis"
+)
+
+var namespacePrefix string
+
+type Config struct {
+	Namespace                 bool          `yaml:"namespace"`
```

Comment:
will modify cntrl to namespace and this to instance for individual configurability